drivers/net/bnxt/bnxt_hwrm.c

```diff
@@ -2007,3 +2007,21 @@ int bnxt_hwrm_pf_evb_mode(struct bnxt *bp)
 
 	return rc;
 }
+
+int bnxt_hwrm_func_bw_cfg(struct bnxt *bp, uint16_t vf,
+			uint16_t max_bw, __rte_unused uint16_t min_bw, uint16_t enables)
+{
```

Comment:
I don't see any value in passing enables in here.  There's only one value that makes sense for the request fields that are filled in.